cli/src/pcluster/validators/imagebuilder_validators.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+from common.boto3.ec2 import Ec2Client
+from pcluster import utils
+from pcluster.models.common import FailureLevel, Param, Validator
+
+
+class AMIVolumeSizeValidator(Validator):
+    """AMI root volume size validator."""
+
+    def _validate(self, volume_size: Param, image: Param):
+        """Validate the volume size is larger than base ami volume size."""
+        ami_id = utils.evaluate_ami_id(image.value)
+        ami_info = Ec2Client().describe_image(ami_id)
+        reserved_size = 15
+        base_ami_volume_size = ami_info.get("BlockDeviceMappings")[0].get("Ebs").get("VolumeSize")
+        min_volume_size = base_ami_volume_size + reserved_size
+        if volume_size.value < base_ami_volume_size:
```

Comment:
It's possible to have one failure with `volume_size.value < min_volume_size`
"Root volume size (size) is lesser than the minimum required size of base ami (size) plus size (15) to allow PCluster software stack installation"?

In this way we are clear from the first error on what value we are expecting.

Reply:
Done